pkgs/servers/bazarr/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, makeWrapper, python3, python3Packages, ... }:
```

Comment:
python3Packages is unused, and the `...` should be removed (and replaced with explicit dependencies, if needed).